kubevirtci/cluster-up/cluster/k8s-provider-common.sh

```diff
@@ -39,6 +39,22 @@ function wait_for_kwok_ready() {
     fi
 }
 
+function configure_nfs() {
+    if [[ "$KUBEVIRT_DEPLOY_NFS_CSI" == "true" ]] && [[ -n "$KUBEVIRT_NFS_DIR" ]]; then
+        ${_cri_bin} run --privileged --rm -v /:/hostroot \
+            --entrypoint /bin/sh ${_cli_container} \
+            -c "mkdir -p /hostroot/${KUBEVIRT_NFS_DIR} && chmod 777 /hostroot/${KUBEVIRT_NFS_DIR}"
+        for disk in disk1 disk2 disk3 disk4 disk5 disk6 disk7 disk8 disk9 disk10 extraDisk1 extraDisk2; do
+            ${_cri_bin} run --privileged --rm -v $KUBEVIRT_NFS_DIR:/nfsdir \
+                --entrypoint /bin/sh ${_cli_container} \
+                -c "mkdir -p /nfsdir/${disk} && chmod 777 /nfsdir/${disk}"
+        done
+        ${_cri_bin} run --privileged --rm -v .:/stuff \
+            --entrypoint /bin/sh ${_cli_container} \
+            -c "mkdir -p /hostroot/${KUBEVIRT_NFS_DIR} && chmod 777 /hostroot/${KUBEVIRT_NFS_DIR}"
```

Comment:
This doesn't start an NFS server does it?

Reply:
The NFS server is started when passing "--nfs-data" to gocli.
These last 3 lines are leftover crap, will remove for next test.
The lines before that populate the NFS server, similar to https://github.com/kubevirt/kubevirtci/blob/29ba6f0ee9a92d984238f9c8e7c93cdaa0485a4e/cluster-provision/gocli/opts/nfscsi/manifests/nfs-server.yaml#L57-L81

Reply:
> The NFS server is started when passing "--nfs-data" to gocli.

may be worth a small note